### DISCUSSION TITLE
Messenger not compatible

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#524] [Q&A]
Why everything say that Fossify Messenger is not compatible with my 5 yo tablet? It's a Lenovo Yoga Smart Tab YT-X705L and runs factory Android 10 atm. Both Droid-ify and F-droid apps say that Messenger isn't compatible with the device while f-droid.org says it requires Android 6.0+. Thats the only Fossify app having such issue. Why?

**Comment by Aga-C** [comment]
Does it work when you install directly from an APK file, e.g. from here: https://github.com/FossifyOrg/Messages/releases/tag/1.0.1?

**Reply by rafnov** [reply]
Yes, I was able to install and run it directly from app page.

**Comment by naveensingh** [comment]
Also, does your tablet support SIM cards? Fossify Messages requires telephony features.

**Reply by rafnov** [reply]
Yes, model X705L supports telephony and I am able to receive SMS to Quik.

**Reply by naveensingh** [reply]
Besides telephony, Fossify Messages doesn't strictly request any other features. This sounds like a bug in the F-Droid app but I am not sure.

**Comment by naveensingh** [comment]
@rafnov Are you still facing this issue?

**Comment by rafnov** [comment]
No. It works since some time.
